gadgets/trace_open/test/trace_open_test.go

```diff
@@ -46,20 +47,36 @@ type traceOpenEvent struct {
 func TestTraceOpen(t *testing.T) {
 	gadgettesting.RequireEnvironmentVariables(t)
 
-	cn := "test-trace-open"
-	containerFactory, err := containers.NewContainerFactory("docker")
+	runtime := "docker"
```

Comment:
I suppose it's fairly simple to allow this to be configurable (in another PR for sure). Using an env variable seems the easiest way, right?

Reply:
Yep, that's why I made that change. To prepare everything to make it easily configurable.

Reply:
I'm still not sure if I prefer using env variable to pass this kind of configuration of flags, like [here](https://github.com/inspektor-gadget/inspektor-gadget/blob/d54838472c7a2be0894f90369bb3068e630415d9/integration/ig/non-k8s/integration_test.go#L32).

Reply:
I think the flag doesn't work well as we'd need to have a `TestMain` function to parse the flag, then running the test alone (in the gadget folder) won't work.